packages/driver/src/cypress/runner.ts

```diff
@@ -987,6 +987,8 @@ const _runnerListeners = (_runner, Cypress, _emissions, getTestById, getTest, se
       hookName = getHookName(runnable)
       const test = getTest() || getTestFromHookOrFindTest(runnable)
 
+      const unsupportedPlugin = runnable.invocationDetails && runnable.invocationDetails.originalFile && runnable.err && runnable.err.message ? $errUtils.getUnsupportedPlugin(runnable.invocationDetails.originalFile, runnable.err.message) : null
```

Comment:
Maybe pass the `runnable` to `$errUtils.getUnsupportedPlugin` and let it take care of this logic and return null if these properties aren't available?